client/my-sites/themes/theme-showcase.jsx

```diff
@@ -388,40 +384,6 @@ class ThemeShowcase extends Component {
 		this.scrollToSearchInput();
 	};
 
-	onDesignYourOwnClick = () => {
-		const { isLoggedIn } = this.props;
-
-		recordTracksEvent( 'calypso_themeshowcase_pattern_assembler_top_button_click', {
-			is_logged_in: isLoggedIn,
-		} );
-
-		this.onDesignYourOwnCallback();
-	};
-
-	onDesignYourOwnCallback = () => {
```

Comment:
There are still multiple occurrences of `this.onDesignYourOwnCallback` in this file; let's also clean them up.